jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -52,7 +53,7 @@ When you choose btn:[Commit and Run], as described above, a pipeline is triggere
 
 If you choose to review the configuration file in the config editor first, the pipeline is triggered after you save the `.circleci/config.yml` by clicking the btn:[Commit and Run] button.
 
-Each time you push changes to your GitHub repository, a new pipeline is triggered. You will see the pipeline running in the CircleCI web app.
+Upon xref:github-app-trigger-options/#supported-trigger-options[the event you selected under the "Run on" menu], a new pipeline is triggered. You will see the pipeline running in the CircleCI web app.
```

Comment:
THis whole Trigger a pipeline in CircleCI section seems wrong now, for example "When you choose btn:[Commit and Run], as described above" ... I don't think we do describe it above anymore, could you check and see what you think? The section can probably be simplified with a link to how to trigger or something, or be condensed into the section above.

Reply:
I added a line in the "When you create a new organization, you will be guided to start creating projects, as follows:" bullet point list to mention the "new" `Commit config and run` button